### ISSUE TITLE
[SOLIDITY SUMMIT ] Last Steps to be production-complete

### ISSUE THREAD

**Issue by geleeroyale** [interspacechat/interspace.chat#50]
- [x] production frontend deployed
- [x] production jitsi server deployed
- [x] disperse admin tokens
- [x] graphic for youtube event: **Stream Title**
- [x] graphic for youtube stream: **Stay Tuned Image**
- [x] youtube events created
- [x] add help for solidity-summit
- [x] edit help
- [x] change about to embed main solidity-summit page
- [x] change credits to display in floating window ~~usual collapsible~~
- [x] update credits.js
- [x] change livestream.js to final stream link
- [x] implement mobile view
- [x] remove hover filter for titles
- [x] update main element size

**Comment by franzihei** [comment]
I will send you the graphics shortly (for YouTube). :)

**Comment by franzihei** [comment]
Screen for before conference starts and breaks see here: https://docs.google.com/presentation/d/1swV9wFuixXZO9uYXZV5QLr6_sBwqxZx2TSPow17AM9o/edit?usp=sharing

**Comment by geleeroyale** [comment]
> Screen for before conference starts and breaks see here: 

For the pause stream the streamer can have the animation open in another tab - otherwise we can't export the animation I think.

**Comment by franzihei** [comment]
![SolSummit_Livestream_Day1](https://user-images.githubusercontent.com/41991517/80404768-f644ef80-88c1-11ea-879a-d47e290bf4c6.png)
![SolSummit_Livestream_Day2](https://user-images.githubusercontent.com/41991517/80404772-f6dd8600-88c1-11ea-981b-5a9c1f28043f.png)

**Comment by franzihei** [comment]
For the breaks can't I just also share my screen with this animation? I'll be moderating the two days anyways

**Comment by franzihei** [comment]
I just realized the `Solidity Summit` was weirdly moved to the left, so here again centered: 

![Bildschirmfoto 2020-04-27 um 20 25 45](https://user-images.githubusercontent.com/41991517/80407113-62752280-88c5-11ea-90b5-90d60411b16e.png)


![Bildschirmfoto 2020-04-27 um 20 25 34](https://user-images.githubusercontent.com/41991517/80407104-5ee19b80-88c5-11ea-87a2-2a7140ab4667.png)

**Comment by geleeroyale** [comment]
> For the breaks can't I just also share my screen with this animation? I'll be moderating the two days anyways (in case that makes things easier that is)

That's true, but you also should enjoy your breaks! It has to be done by the streamer, because the sound should be faded out with the picture - if you do it, all sound would still be there and the streamer will be less attentive because he does not have to do anything :)

(thanks for the title cards - I deleted your first comment with the pics so its clear which to use)
